Exiled.API/Features/Toys/Primitive.cs

```diff
@@ -95,6 +148,7 @@ public static Primitive Create(Vector3? position = null, Vector3? rotation = nul
                 primitive.Spawn();
 
             primitive.AdminToyBase.NetworkScale = primitive.AdminToyBase.transform.localScale;
+            primitive.Color = color ?? Color.white;
```

Comment:
```suggestion
            primitive.Color = color ?? Color.gray;
```
Should match NW's default